docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,597 @@
+
+**Note: This example is for illustration purposes only. Obviously, in a production environment you would never use `tls-admin` and `tls-adminpw` as the bootstrap username and password.**  Be sure that you record the admin id and password that you specify. They are required later when you issue enroll commands against the CA. It can help to use a meaningful id to differentiate which server you are transacting with and follow secure password practices.  
+
+#### What does the CA server `init` command do?
+
+The `init` command does not actually start the server but generates the required metadata if it does not already exist for the server:
+- Sets the default the CA Home directory (referred to as `FABRIC_CA_HOME` in these instructions) to where the `fabric-ca-server init` command is run.
+- Generates the default configuration file `fabric-ca-server-config.yaml`  which can be used as a template for your server the `FABRIC_CA_HOME` directory. We refer to this file throughout these instructions as the "configuration .yaml" file.
+- Creates the root certificate file `ca-cert.pem`, if it does not already exist in the CA Home directory. This is the **self-signed root certificate**, meaning it is generated and signed by the CA itself and does not come from another source. This root cert is the public key of the CA that allows other organizations to verify that your peers and applications are valid participants.
+- Generates the CA server private key and stores it in the `FABRIC_CA_HOME` directory under `/msp/keystore`.
+- Initializes a default sqlLite database for the server although you can modify the database setting in the configuration .yaml file to use the supported database of your choice. Every time the server is started, it loads the data from this database.
+- Bootstraps the CA server administrator, specified by the `-b` flag parameters `<ADMIN_USER>` and `<ADMIN_PWD>`, onto the server. The admin user is registered with all of the default admin attributes present in the configuration .yaml file `registry` section. If this CA will be used to register other users with any of those attributes, then the CA admin user needs to possess those attributes. In other words, the registrar must have the `hf.Registrar.Roles` attributes before it can register another identity with any of those attributes. Therefore, if this CA admin will be used to register the admin identity for an Intermediate CA, then this CA admin must have the `hf.IntermediateCA` set to `true` even though this may not be an intermediate CA server. The default settings already include these attributes.
```

Comment:
This says the default admin attributes from the config file will be used, however bullet 2 above says the config file is generated at the same time.  Are you suggesting that a config file should exist prior to init?

Reply:
this comment is still outstanding.

Reply:
Good point. I changed it to:
When the CA server is subsequently started, the admin user is registered with the admin attributes provided in the configuration .yaml file `registry` section.